content/en/gitops.md

```diff
@@ -1,29 +1,38 @@
 ---
 title: GitOps
-status: Feedback Appreciated
+status: Completed
 category: Concept
 tags: ["methodology", "", ""]
 ---
 
-GitOps is a set of best practices based on [shared principles](https://opengitops.dev/), 
-applied to a workflow that depends on software agents that 
-enable automation to reconcile a declared system state or configuration in a git repository.
-These software agents and practices are used to execute a cohesive workflow that 
+GitOps is a set of best practices for managing software systems  
+from the beginning of the development workflow, all the way to deployment
```

Comment:
```suggestion
from the beginning of the development workflow, all the way to deployment.
```

Reply:
I would change this to say "In a continuous manner" as the hallmark of GitOps is it's continuous.